src/containers/Layout/Layout.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Footer from '../../components/Layout/Footer/Footer'
+import Header from '../../components/Layout/Header/Header'
+import PropTypes from 'prop-types'
+
+export default class Layout extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      mobileMenu: false
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
```

Comment:
Let's get rid of the constructor, and use the newer state initializer syntax, with or without the semicolon, as so:
<img width="379" alt="Screen Shot 2019-04-26 at 11 21 15 PM" src="https://user-images.githubusercontent.com/11988089/56878542-2609af00-6a44-11e9-8c37-237dec1867f7.png">